### ISSUE TITLE
false positive on indent formatting

### ISSUE THREAD

**Issue by IvanPizhenko** [pinterest/ktlint#1837]
## Expected Behavior

Formatting is expected to be valid. 

## Observed Behavior
IDE shows formatting as valid, but check with Gradle fails. Following errors occur:
```
$ ./gradlew ktlintCheck
...
myfile.kt:33:1 Unexpected indentation (8) (should be 6)
myfile.kt:34:1 Unexpected indentation (8) (should be 6)
myfile.kt:35:1 Unexpected indentation (8) (should be 6)
```

## Steps to Reproduce

Code:
```kotlin
    val flag = response.isSuccess() &&
      someService?.someMethod(
        parameter1 = response.body().someObject, // line 33
        parameter2 = LocalDate.parse(ctx.formParam("someParameter")),
      ) == true
```
Settings: All indents are set to 2 spaces (see editorconfig below).

## Your Environment

```
Ubuntu 22.04

Gradle 8.0.1

Ktlint plugin:
  id("org.jlleitschuh.gradle.ktlint") version "11.2.0" apply false

IntelliJ:
IntelliJ IDEA 2022.3.2 (Community Edition)
Build #IC-223.8617.56, built on January 26, 2023
Runtime version: 17.0.5+1-b653.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.15.0-60-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 2000M
Kotlin: 223-1.8.0-release-345-IJ8617.56
Non-Bundled Plugins:
    org.rust.lang (0.4.188.5205-223)
    idea.plugin.protoeditor (223.8214.6)
    com.gmike.visual-studio-2019-dark (1.5.3)
    com.samdark.intellij-visual-studio-code-dark-plus (2.6)
    com.nbadal.ktlint (0.12.0)
    org.jusecase.jte-intellij (2.0.9)
    zielu.gittoolbox (500.0.4+213)
    com.dt.theme.vs (1.0)
    aws.toolkit (1.61-223)

```

editorconfig
```
[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
insert_final_newline = true
max_line_length = 120
tab_width = 2
trim_trailing_whitespace = true
ij_formatter_off_tag = @formatter:off
ij_formatter_on_tag = @formatter:on
ij_formatter_tags_enabled = true
ij_smart_tabs = false
ij_visual_guides = none
ij_wrap_on_typing = false

[{*.kt,*.kts}]
ij_continuation_indent_size = 2
ij_kotlin_align_in_columns_case_branch = false
ij_kotlin_align_multiline_binary_operation = false
ij_kotlin_align_multiline_extends_list = false
ij_kotlin_align_multiline_method_parentheses = false
ij_kotlin_align_multiline_parameters = true
ij_kotlin_align_multiline_parameters_in_calls = false
ij_kotlin_allow_trailing_comma = true
ij_kotlin_allow_trailing_comma_on_call_site = true
ij_kotlin_assignment_wrap = normal
ij_kotlin_blank_lines_after_class_header = 0
ij_kotlin_blank_lines_around_block_when_branches = 0
ij_kotlin_blank_lines_before_declaration_with_comment_or_annotation_on_separate_line = 1
ij_kotlin_block_comment_at_first_column = true
ij_kotlin_call_parameters_new_line_after_left_paren = true
ij_kotlin_call_parameters_right_paren_on_new_line = true
ij_kotlin_call_parameters_wrap = on_every_item
ij_kotlin_catch_on_new_line = false
ij_kotlin_class_annotation_wrap = split_into_lines
ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
ij_kotlin_continuation_indent_for_chained_calls = false
ij_kotlin_continuation_indent_for_expression_bodies = false
ij_kotlin_continuation_indent_in_argument_lists = false
ij_kotlin_continuation_indent_in_elvis = false
ij_kotlin_continuation_indent_in_if_conditions = false
ij_kotlin_continuation_indent_in_parameter_lists = false
ij_kotlin_continuation_indent_in_supertype_lists = false
ij_kotlin_else_on_new_line = false
ij_kotlin_enum_constants_wrap = off
ij_kotlin_extends_list_wrap = normal
ij_kotlin_field_annotation_wrap = split_into_lines
ij_kotlin_finally_on_new_line = false
ij_kotlin_if_rparen_on_new_line = true
ij_kotlin_import_nested_classes = false
ij_kotlin_imports_layout = *
ij_kotlin_insert_whitespaces_in_simple_one_line_method = true
ij_kotlin_keep_blank_lines_before_right_brace = 1
ij_kotlin_keep_blank_lines_in_code = 1
ij_kotlin_keep_blank_lines_in_declarations = 1
ij_kotlin_keep_first_column_comment = true
ij_kotlin_keep_indents_on_empty_lines = false
ij_kotlin_keep_line_breaks = true
ij_kotlin_lbrace_on_next_line = false
ij_kotlin_line_comment_add_space = false
ij_kotlin_line_comment_at_first_column = true
ij_kotlin_method_annotation_wrap = split_into_lines
ij_kotlin_method_call_chain_wrap = normal
ij_kotlin_method_parameters_new_line_after_left_paren = true
ij_kotlin_method_parameters_right_paren_on_new_line = true
ij_kotlin_method_parameters_wrap = on_every_item
ij_kotlin_name_count_to_use_star_import = 2147483647
ij_kotlin_name_count_to_use_star_import_for_members = 2147483647
ij_kotlin_parameter_annotation_wrap = off
ij_kotlin_space_after_comma = true
ij_kotlin_space_after_extend_colon = true
ij_kotlin_space_after_type_colon = true
ij_kotlin_space_before_catch_parentheses = true
ij_kotlin_space_before_comma = false
ij_kotlin_space_before_extend_colon = true
ij_kotlin_space_before_for_parentheses = true
ij_kotlin_space_before_if_parentheses = true
ij_kotlin_space_before_lambda_arrow = true
ij_kotlin_space_before_type_colon = false
ij_kotlin_space_before_when_parentheses = true
ij_kotlin_space_before_while_parentheses = true
ij_kotlin_spaces_around_additive_operators = true
ij_kotlin_spaces_around_assignment_operators = true
ij_kotlin_spaces_around_equality_operators = true
ij_kotlin_spaces_around_function_type_arrow = true
ij_kotlin_spaces_around_logical_operators = true
ij_kotlin_spaces_around_multiplicative_operators = true
ij_kotlin_spaces_around_range = false
ij_kotlin_spaces_around_relational_operators = true
ij_kotlin_spaces_around_unary_operator = false
ij_kotlin_spaces_around_when_arrow = true
ij_kotlin_variable_annotation_wrap = off
ij_kotlin_while_on_new_line = false
ij_kotlin_wrap_elvis_expressions = 1
ij_kotlin_wrap_expression_body_functions = 1
ij_kotlin_wrap_first_method_in_call_chain = false

# jOOQ generated classes
[**/src/main/kotlin/**/generated/**/*.kt]
ktlint_disabled_rules=colon-spacing,import-ordering,no-consecutive-blank-lines,no-semi
max_line_length=off

# test properties files
[**/{test,testFixtures}/**/*.properties]
trim_trailing_whitespace = false
```

**Comment by paul-dingemans** [comment]
I can *not* reproduce the problem with Ktlint CLI `0.48.2` with code sample below and with your `.editorconfig` configuration above:
```
class Foo {
  fun bar() {
    val flag = response.isSuccess() &&
      someService?.someMethod(
        parameter1 = response.body().someObject, // line 33
        parameter2 = LocalDate.parse(ctx.formParam("someParameter")),
      ) == true
  }
}
```
Can you try to reproduce the problem with Ktlint CLI (see https://pinterest.github.io/ktlint/install/cli/) on your code base?

**Comment by IvanPizhenko** [comment]
I am not getting this with ktlint CLI, but I'm still getting this with Gradle plugin. Maybe Gradle plugin uses not the latest ktlint CLI version. Need to check.
